lib/commons/forms.less

```diff
@@ -83,23 +83,20 @@ textarea,
     color: @disabled;
   }
 
-  &@{prefix}required {
-    &:after {
-      content: 'Required';
-      display: inline-block;
-      margin-left: @space-large;
-      font-style: italic;
-      font-weight: @weight-normal;
-    }
-  }
-
   &@{prefix}error {
```

Comment:
Thanks. I made this change by adding the `error` class to `required` and moving my earlier code.